src/common/state/redis.c

```diff
@@ -262,63 +262,6 @@ void db_attach(db_handle *db, event_loop *loop, bool reattach) {
   }
```

Comment:
Would you mind also removing the `REDIS_MULTI_CALLBACK_HEADER` with this commit? I think it's obsolete now :)